Formula/influxdb.rb

```diff
@@ -111,9 +123,11 @@ def install
 
     cd influxdb_path do
       if build.head?
-        system "go", "install", "-ldflags", "-X main.version=0.14.0-HEAD -X main.branch=master -X main.commit=#{`git rev-parse HEAD`.strip}", "./..."
+        system "go", "install", "-ldflags", "-X main.version=1.0.0-HEAD -X main.commit=#{`git rev-parse HEAD`.strip}", "./..."
+      elsif build.devel?
+        system "go", "install", "-ldflags", "-X main.version=1.0.0-beta2 -X main.commit=bed66cfbe4abf7558594a86f1779c809ffe23b1d", "./..."
```

Comment:
If we built this from the Git repo could we parse `bed66cfbe4abf7558594a86f1779c809ffe23b1d` automatically? That might be preferable.